src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState('');
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      setHasError('');
+
+      try {
+        const data = await getPeople();
+
+        setPeople(data);
+      } catch {
+        setHasError('something went wrong');
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    fetchPeople();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {isLoading ? (
+            <Loader />
+          ) : (
+            <>
+              {hasError && (
+                <p data-cy="peopleLoadingError" className="has-text-danger">
+                  {hasError}
+                </p>
+              )}
+
+              {!people.length && (
```

Comment:
<img width="963" alt="image" src="https://github.com/mate-academy/react_people-table-basics/assets/86789161/ce639353-5f48-4102-8903-a156d75f7022">

2 messages are shown at the same time